l10n_br_account_product/models/account_invoice.py

```diff
@@ -756,6 +757,21 @@ def _compute_price(self):
     freight_value = fields.Float(
         'Frete', digits=dp.get_precision('Account'), default=0.00)
     fiscal_comment = fields.Text(u'Observação Fiscal')
+    xped = fields.Char(
+        string=u"Código do Pedido (xPed)",
+        size=15,
+    )
+    nitemped = fields.Char(
+        string=u"Item do Pedido (nItemPed)",
+    )
+
+    @api.onchange("nitemped")
```

Comment:
Why not use Integer field for that? And why just this field is important to have a specific onchange, the user can type wrong values in any field.